recipes-ni/ni-grpc-device/ni-grpc-device_1.0.1.bb

```diff
@@ -0,0 +1,40 @@
+SUMMARY = "gRPC server providing remote access to NI device driver APIs."
+DESCRIPTION = ""
+HOMEPAGE = "https://github.com/ni/grpc-device"
+SECTION = "base"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "\
+	https://github.com/ni/grpc-device/releases/download/${${PN}_RELEASE_TAG}/ni-grpc-device-server-ni-linux-rt-x64.tar.gz;name=release-tar;downloadfilename=${PN}_${${PN}_RELEASE_TAG}.tar.gz;subdir=${PN}-server \
+	https://github.com/ni/grpc-device/releases/download/${${PN}_RELEASE_TAG}/ni-grpc-device-client.tar.gz;name=release-client-tar;downloadfilename=${PN}-client_${PN}_${${PN}_RELEASE_TAG}.tar.gz;subdir=${PN}-client \
+"
+
+SRC_URI[release-tar.md5sum] = "200de2f8b1cbc4c26af42cfaed1b01ea"
+SRC_URI[release-tar.sha256sum] = "32e0767b28f71a3024921904aa7bcd879cf9e9f2a98661722ac549446bca0aed"
+SRC_URI[release-client-tar.md5sum] = "11da718d6be42848a1e4155e56bdb641"
+SRC_URI[release-client-tar.sha256sum] = "0555fc1f2867b4ed9314354ed87041b7065365a24ddaf3526c1871f519df34ad"
+
+${PN}_RELEASE_TAG ?= "v${PV}"
+${PN}_RELEASE_TAG = "v${PV}-rc1"
+
+S = "${WORKDIR}"
+
+DEPENDS += "\
+"
+
+RDEPENDS_${PN} += "\
+	python3 \
+"
+
+do_install_append () {
+	install -d ${D}${bindir}
+	install --mode=0755 --owner=0 --group=0 ${S}/${PN}-server/ni_grpc_device_server ${D}${bindir}
```

Comment:
Why 0 for owner and group?

Reply:
To assert that it should be owned by the root user. I'm not 100% sure I'm required to specify the ownership here (0:0 might be the default), but I wanted to be explicit.

Reply:
Ok, I assume the ownership specs have been confirmed with the Sync team.